src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,24 @@
 package mate.jdbc;
 
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Manufacturer;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final String PACKAGE_NAME = "mate.jdbc";
+    private static final Injector injector = Injector.getInstance(PACKAGE_NAME);
 
+    public static void main(String[] args) {
+        ManufacturerDao manufacturerDao = (ManufacturerDao) injector
+                .getInstance(ManufacturerDao.class);
+        //testing:
+        manufacturerDao.create(new Manufacturer("audi", "Germany"));
+        manufacturerDao.create(new Manufacturer("infiniti", "Japan"));
+        manufacturerDao.create(new Manufacturer("Ferrari", "Italy"));
+        manufacturerDao.create(new Manufacturer("Cadillac", "USA"));
+        manufacturerDao.delete(12L);
```

Comment:
```suggestion
        Manufacturer cadillac = manufacturerDao.create(new Manufacturer("Cadillac", "USA"));
        manufacturerDao.delete(cadillac.getId());
```
let's use `manufacturer.getId()` instead of `12L`
for example, if I run this code on my laptop it will not work. I don't have manufacturer with such id.